connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginScannerTest.java

```diff
@@ -137,6 +139,17 @@ public void testVersionedPluginsHasVersion(PluginScanner scanner) {
         versionedPluginResult.forEach(pluginDesc -> assertEquals("1.0.0", pluginDesc.version()));
     }
 
+    @ParameterizedTest
+    @MethodSource("parameters")
+    public void testClasspathPluginIsAlsoLoadedInIsolation(PluginScanner scanner) {
+        // json converter is part of the classpath by default
+        String jsonConverterLocation = JsonConverter.class.getProtectionDomain().getCodeSource().getLocation().getPath();
```

Comment:
This feels very brittle to me.

Rather than using a real class, can this use a TestPlugin with a similar name to a classpath plugin?

Or the last time we needed to set up a "classpath" plugin, it was by injecting a new loader in the hierarchy in `PluginsTest#assertClassLoaderReadsVersionFromResource`.